packages/bpk-component-bottom-sheet/src/BpkBottomSheet-test.tsx

```diff
@@ -85,17 +91,39 @@ describe('BpkBottomSheet', () => {
   it('renders correctly with action props', () => {
     const { asFragment } = render(
       <BpkBottomSheet
-        ariaLabelledby='bottom-sheet'
+        {...props}
         actionText='Action'
-        id="my-bottom-sheet"
-        isOpen
         onAction={jest.fn()}
-        onClose={jest.fn()}
         wide
       >
         Bottom Sheet content
       </BpkBottomSheet>
     );
     expect(asFragment()).toMatchSnapshot();
   });
+  it('renders correctly with ariaLabelledBy prop', () => {
+    const { container } = render(
+      <BpkBottomSheet
+        {...props}
+        ariaLabelledby='my-bottomsheet-title-id'
+      >
+        Bottom Sheet content
+      </BpkBottomSheet>
+    );
+
+    expect(container.querySelector('dialog[aria-labelledby="my-bottomsheet-title-id"]')).toBeInTheDocument();
+  });
+  it('renders correctly with ariaLabel prop', () => {
+    const { container } = render(
+      <BpkBottomSheet
+        {...props}
+        ariaLabel='my a11y title'
```

Comment:
I'm guessing this will take precedence since it's added after `ariaLabelledBy`? 🤔

Reply:
I have done some testing in Safari with VoiceOver and if you give it a valid `ariaLabelledBy` then it will use that. If the labelledBy was missing or invalid then it falls back to the `aria-label` instead.